components/stage/scripts/create.py

```diff
@@ -22,6 +22,8 @@ def install_stage():
 
     nodejs_source_url = ctx_properties['nodejs_tar_source_url']
     stage_source_url = ctx_properties['stage_tar_source_url']
+    print "stage_source_url={0}".format(stage_source_url)
+    ctx.instance.runtime_properties['ignore_ui'] = 'False'
```

Comment:
Perhaps call the runtime property: "skip_stage_installation" and set it to 'true'